lib/Cleantalk/ApbctWP/CleantalkRealPerson.php

```diff
@@ -30,7 +30,18 @@ public function __construct()
     public function publicCommentAddTrpClass($classes, $_css_class, $comment_id, $comment, $_post)
     {
         $ct_hash = get_comment_meta((int)$comment_id, 'ct_real_user_badge_hash', true);
-        if ( $ct_hash && $comment->comment_author ) {
+
+        $show_trp = false;
+        $roles_to_show_trp = ['administrator', 'editor'];
+        // Check user role for show TRP
+        if ( $comment->user_id ) {
+            $user = get_userdata($comment->user_id);
+            if ( $user && is_array($user->roles) && count(array_intersect($user->roles, $roles_to_show_trp)) > 0 ) {
+                $show_trp = true;
+            }
+        }
+
+        if ( ( $ct_hash && $comment->comment_author ) || $show_trp ) {
```

Comment:
How about predeclaring (bool)$show_trp_on_roles and (bool)$show_trp_on_hash?

This is more readable i presume.

```
if ($show_trp_on_roles || $show_trp_on_hash) {
    $classes[] = 'apbct-trp';
    return $classes;
}
```

Reply:
Good idea! Done https://github.com/CleanTalk/wordpress-antispam/commit/76ffab0bb634dcc00e4fbf22e9e756b6c2b2fa9b